projects/storefrontlib/cms-structure/page/slot/page-slot.component.spec.ts

```diff
@@ -332,7 +332,7 @@ describe('PageSlotComponent', () => {
       fixture.detectChanges();
       pageSlotComponent.getComponentDeferOptions('CMSTestComponent');
       expect(pageSlotService.getComponentDeferOptions).toHaveBeenCalledWith(
-        undefined,
+        '',
```

Comment:
Was there an alternative to changing this behaviour?

Reply:
revert back